### ISSUE TITLE
Feature Request: Allow specifying multiple creators to scrape

### ISSUE THREAD

**Issue by AlexDucks** [DIGITALCRIMINAL/OnlyFans#1203]
In case it wasn't clear - when `auto_model_choice` is set to `false` - allowing you to specify a "list" (e.g. space-separated) of creators to scrape would be handy.

Right now my method is scraping a profile one-by-one (I'm subscribed to 50+, going through them all takes a while, so I just manually update those I really want fully up-to-date), but it's a bit tedious when you have too many subscriptions.

Say you have the following options:  
`Names: Username = username  | 0 = All | 1 = megturney | 2 = omgcosplay | 3 = peachjars | 4 = darshelle`

Typing `1 4` (1 + 4) should then scrape the profiles: `megturney` and `darshelle`

Bonus points if it also supports the current functionality of specifying usernames instead of numbers ("`megturney darshelle`").

Thanks :)

**Comment by DIGITALCRIMINAL** [comment]
You can do already but I haven't documented it. 
You can change auto_model_choice into an array and pass username or number.

You can do this for all auto choices except for auto_api_choice since that uses a different method atm

**Comment by neuri4848** [comment]
1,4 specifies meg turney and darshelle

**Comment by AlexDucks** [comment]
> You can do this already but I haven't documented it.
> You can change auto_model_choice into an list and pass usernames and numbers.
> 
> You can do this for all auto choices except for auto_api_choice since that uses a different method atm.
> 
> choice = ["model_1","model_2"]

Good to know, but wasn't exactly what I requested, since I'd rather not have to edit the configuration file constantly (though handy if I wanna scrape many, but not all).

Someone else commented (I got an email) that it was already possible to put `1,4`, but I can't see their comment here. It does seem to work with `megturney,darshelle` as well.  
Not sure why I didn't think to test with a comma separating them, but that's essentially what I wanted, so I'll close the issue.

Thanks for the reply :)
